### PR TITLE
Prepare v2.3.1 release

### PR DESCRIPTION
**Pull request by guihkx**
After this gets merged, there will be some other tasks to do as well.

Check out issue #701 for more info on that.

### CODE REVIEW

File: packaging/linux/common/notes.appdata.xml.in [hunk 1]
```diff
@@ -56,7 +56,18 @@
     </screenshots>
     <content_rating type="oars-1.1" />
     <releases>
-    <release date="2024-08-31" version="2.3.0">
+        <release date="2024-09-30" version="2.3.1">
+            <description>
```
Comment (guihkx) [inline packaging/linux/common/notes.appdata.xml.in:60]: @nuttyartist Please let me know if this change log is written to your liking.

**Comment by guihkx** [comment]
Force-pushed to sync with `master`.

File: packaging/linux/common/notes.appdata.xml.in (hunk 1 above)
Comment (nuttyartist) [reply packaging/linux/common/notes.appdata.xml.in:60]: Looks good to me.

File: packaging/windows/Notes_Inno_Script_Github.iss [hunk 2]
```diff
@@ -8,9 +8,9 @@ AppId={{3C782395-4B68-4EA5-A2F4-C60E886CD71F}
 AppName=Notes
 AppVersion={#Version}
 AppPublisher=Awesomeness
-AppPublisherURL=https://www.get-notes.com/
-AppSupportURL=https://www.get-notes.com/
-AppUpdatesURL=https://www.get-notes.com/
+AppPublisherURL=https://www.notes-foss.com/
```
Comment (nuttyartist) [inline packaging/windows/Notes_Inno_Script_Github.iss:11]: 👍

File: packaging/linux/debian/changelog.in [hunk 3]
```diff
@@ -4,6 +4,13 @@
 
  -- @APP_AUTHOR@ <@APP_AUTHOR_EMAIL@>  @DATE_NOW_UTC@
 
+@PROJECT_NAME_LOWERCASE@ (2.3.1) stable; urgency=low
+
+  * Fixed a visual bug where the "Hide to tray" option would not reflect its real state.
```
Comment (nuttyartist) [inline packaging/linux/debian/changelog.in:9]: 👍

**Review by nuttyartist — APPROVED** [review]

**Comment by guihkx** [comment]
Thanks for reviewing! I'll just bump the release date and merge this.

**Pull request merged**